demux/demux_lavf.c

```diff
@@ -183,6 +183,12 @@ static const struct format_hack format_hacks[] = {
     {0}
 };
 
+typedef struct {
+    int num;
+    int pcr_pid;
+    unsigned int nb_stream_indexes;
+} lavf_program_t;
```

Comment:
New mpv code doesn't do these typedefs anymore. Using an non-typedefed struct type is preferred. Also, `_t` suffixes are reserved by POSIX (lots of code including old MPlayer code ignores this, but still).